app/mailers/referee_mailer.rb

```diff
@@ -38,6 +38,17 @@ def reference_confirmation_email(application_form, reference)
     )
   end
 
+  def reference_cancelled_email(application_form, reference)
```

Comment:
Could we use `reference.application_form` instead here instead of passing in `application_form`?